### ISSUE TITLE
Can you share the model weights of ImageNet-SCT

### ISSUE THREAD

**Issue by ioslide** [nis-research/nn-frequency-shortcuts#3]
: )

**Comment by ShunxinWang** [comment]
Hi, we don't train models on ImageNet-SCT, which is only used as a test set

**Comment by ShunxinWang** [comment]
If you mean the model weights on ImageNet-10, you can run the following command line for training the model. Using the same hyperparameters, model performance won't deviate much. 
``` 
 python -u train.py   --backbone_model resnet18 --lr 0.01 --dataset imagenet10   --save_dir results/  --image_size 224  --num_class 10    
```
